api-server/routes/session.py

```diff
@@ -61,7 +61,7 @@ async def join(session_id: str, join_body: JoinRequestBody):
         raise HTTPException(
             status.HTTP_404_NOT_FOUND, f"Session {session_id} not found"
         )
-    player_id = f"{session_id}-{join_body.username}"
+    player_id = session_id + "-" + join_body.username
```

Comment:
Any reason for this change? It doesn't matter but just curious. (just ignore it, probably just some issue due to pulling and reorganizing files.)

Reply:
Good catch. Probably something occurred when pulling.